pkg/kv/kvserver/store.go

```diff
@@ -3779,9 +3780,7 @@ func (s *Store) ComputeMetricsPeriodically(
 	// non-periodic callers of this method don't trigger expensive
 	// stats.
 	if tick%logSSTInfoTicks == 1 /* every 10m */ {
-		// NB: The initial blank line ensures that compaction stats display
-		// will not contain the log prefix.
-		log.Storage.Infof(ctx, "\n%s", m.Metrics)
+		log.Storage.Infof(ctx, "Metrics for store %s:\n%s", s.StoreID(), m.Metrics)
```

Comment:
Since we have the NodeID, too, can you also emit the node ID?  Having to work backward in logging systems from the store ID back to a node ID is an extra few steps that we can elide if we emit this value here.

Reply:
Oh!  I missed what `AnnotateCtx()` is actually doing to the output.  If that's the case, why keep this as the original `"\n%s", m.Metrics`?  Our logs are already plenty verbose.
```
		log.Storage.Infof(ctx, "Metrics for node %s store %s:\n%s", s.NodeID(), s.StoreID(), m.Metrics)
```

Reply:
I figured it's better than a blank line, which looks like this:
```
I250218 15:01:57.233455 403 3@kv/kvserver/store.go:3785 ⋮ [T1,Vsystem,n1,s1] 31
I250218 15:01:57.233455 403 3@kv/kvserver/store.go:3785 ⋮ [T1,Vsystem,n1,s1] 31 +      |                             |       |       |   ingested   |     moved    |    written   |       |    amp
```
Maybe I should just put `Pebble metrics:` since that's greppable? In the past, I grepped stuff like `total |` which feels janky.